src/libyang3-py3.patch/0001-debian.patch

```diff
@@ -0,0 +1,68 @@
+diff -ruN libyang-python.orig/debian/changelog libyang-python.new/debian/changelog
+--- libyang-python.orig/debian/changelog	1970-01-01 00:00:00.000000000 +0000
++++ libyang-python.new/debian/changelog	2025-02-09 12:45:51.113010546 +0000
+@@ -0,0 +1,5 @@
++libyang (3.0.3-1) unstable; urgency=low
++
++  * source package automatically created by stdeb 0.10.0
++
++ -- None <None>  Sat, 08 Feb 2025 20:55:02 +0000
+diff -ruN libyang-python.orig/debian/compat libyang-python.new/debian/compat
+--- libyang-python.orig/debian/compat	1970-01-01 00:00:00.000000000 +0000
++++ libyang-python.new/debian/compat	2025-02-09 12:45:51.113010546 +0000
+@@ -0,0 +1 @@
++9
+diff -ruN libyang-python.orig/debian/control libyang-python.new/debian/control
+--- libyang-python.orig/debian/control	1970-01-01 00:00:00.000000000 +0000
++++ libyang-python.new/debian/control	2025-02-09 12:45:51.113010546 +0000
+@@ -0,0 +1,16 @@
++Source: libyang
++Maintainer: None <None>
++Section: python
++Priority: optional
++Build-Depends: dh-python, python3-setuptools, python3-all-dev, python3-cffi, debhelper (>= 9), libyang-dev ( >= 3.0.3)
++Depends: libyang3
```

Comment:
`Depends` should not exist in this section, please remove this line.

Reply:
ok

Reply:
done